tests/gui/actions/test_download.py

```diff
@@ -25,7 +25,8 @@ def on_download_action(state: State):
         pass
 
     # set gui frame
-    gui._set_frame(inspect.currentframe())
```

Comment:
it would have been easier to declare 
`def _set_frame(self, frame: t.Optional[FrameType]):`